tests/unittests/OperatorTest.cpp

```diff
@@ -3314,6 +3314,46 @@ TEST_P(InterpAndCPU, SigmoidCrossEntropyWithLogits) {
   EXPECT_TRUE(expected.isEqual(result->getPayload()));
 }
 
+/// Test the InsertTensor node works correctly.
+TEST_P(InterpAndCPU, insertTensorTest) {
```

Comment:
Question: The PR introduces the new Tile node. Shouldn't we have a test checking that this node is lowered into insert_tensor nodes?

Reply:
Oh, I see in the PR description that we re-use existing tile tests for it.

Reply:
Yeah before this PR there was already a `Graph::createTile()` which has unit tests for it. Before this PR it returned an `InsertTensorNode`, which I now moved to return a `TileNode` which is later lowered to an `InsertNode`.